Library/Homebrew/shims/scm/git

```diff
@@ -99,7 +99,7 @@ brew_repo_version="$(quiet_safe_cd "$SCM_DIR/../../../../bin" && pwd -P)/$SCM_FI
 safe_exec "$brew_repo_version" "$@"
 
 IFS=$'\n'
-for path in $(/usr/bin/which -a "$SCM_FILE" 2>/dev/null)
+for path in $(command -v "$SCM_FILE" 2>/dev/null)
```

Comment:
This patch can go upstream to Homebrew/brew. `which` is not an essential utility on Debian and is not installed by default in the `fedora` Docker image. `command -v` is POSIX and universal.

Reply:
What's a CL? I'd suggest a separate PR for each change.

Reply:
Sorry, yes. "CL" is our jargon at work for PR. You'll see a bunch of small PRs for this.

Reply:
https://github.com/Homebrew/brew/pull/3944